libs/ml/src/dataloaders/tensor_dataloader.cpp

```diff
@@ -49,6 +49,27 @@ typename TensorDataLoader<TensorType>::ReturnType TensorDataLoader<TensorType>::
   return ReturnType(ret_labels, ret_data);
 }
 
+template <typename TensorType>
+OperationsCount TensorDataLoader<TensorType>::ChargeGetNext()
+{
+  OperationsCount cost = 1;
+  for (auto it : one_sample_label_shape_)
+  {
+    cost *= it;
+  }
+  OperationsCount data_cost = 1;
+  for (auto it : one_sample_data_shapes_)
+  {
+    for (auto it2 : it)
+    {
+      data_cost *= it2;
+    }
+  }
+  cost += data_cost;
+  cost += 5;
```

Comment:
add a named constant which better describes where this "5" magic number comes from